spec/lib/formatter/csv/annotation_for_csv_spec.rb

```diff
@@ -233,14 +233,7 @@
       context "with a dropdown task" do
         let(:dd_workflow) { build(:workflow, :complex_task) }
         let(:annotation) do
-          {
-            "task"=>"T7",
-            "value"=>[
-              {"value"=>"c6e0d98477ec8", "option"=>true},
-              {"value"=>"fb39ba165bfd4", "option"=>true},
-              {"value"=>"something unlisted", "option"=>false}
-            ]
-           }
+          {"task"=>"T7", "value"=>[]}
```

Comment:
I don't understand this change. I guess we were missing the test for a blank dropdown annotation? But don't we also want to keep testing a filled-out dropdown annotation?

Reply:
I don't really understand how this data got into our system (possibly the borked workflows bug) but that is an actual value for that workflow in our db. 

There are still tests for filled out dropdown annotations, this change is specifically to handle the bug that i was seeing with unexpected missing annotation values.

Reply:
I might be mistaken, but I thought that uses the annotation you're clearing on the line this comment is attached to, through:
https://github.com/zooniverse/Panoptes/blob/e9a27b9666d746705e495fb686023058c0971829/spec/lib/formatter/csv/annotation_for_csv_spec.rb#L247

Reply:
I think you just need to add a second test for the `it 'works with bad annotations as found in production db'` which uses the empty annotations, while leaving the existing annotations and test alone?

Reply:
thanks for catching this - i completely forgot to add a new it block from my debugging. Absolutely my f*** up

Reply:
No probs.